FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_keep_alive/DemoTasks/KeepAliveMQTTExample.c

```diff
@@ -644,7 +644,7 @@ static void prvUpdateSubAckStatus( MQTTPacketInfo_t * pxPacketInfo )
     for( ulTopicCount = 0; ulTopicCount < ulSize; ulTopicCount++ )
     {
         /* Multiply the index by 2 because the status code consists of two bytes. */
-        xTopicFilterContext[ ulTopicCount ].xSubAckStatus = pucPayload[ ulTopicCount * 2 ];
+        xTopicFilterContext[ ulTopicCount ].xSubAckStatus = pucPayload[ ulTopicCount ];

```

Comment:
The comment and the operation doesn't seems to be matching. Doesn't it  index need to be multiplied? Same comment for all the similar changes in this commit.

Reply:
I talked to Muneeb about this, and the index doesn't need to be multiplied by 2. The reason 2 bytes are added in `MQTT_GetSubAckStatus` is because the packet identifier consists of two bytes.

Reply:
Ok. Sounds good.